cfme/scripting/appliance.py

```diff
@@ -23,11 +21,15 @@ def main():
 
 
 @main.command('reboot', help='Reboots a provider')
-@click.argument('appliance_ip')
-@click.option('--wait_for_ui', is_flag=True, default=True)
+@click.argument('appliance_ip', default=None, required=False)
+@click.option('--wait-for-ui', is_flag=True, default=True)
 def reboot_appliance(appliance_ip, wait_for_ui):
     """Reboots an appliance"""
-    app = IPAppliance(appliance_ip)
+    from utils.appliance import IPAppliance, get_or_create_current_appliance
```

Comment:
lets use a click type or a function for that - the amount of code duplication is no fun